lib/ronin/support/crypto/cert.rb

```diff
@@ -276,6 +277,12 @@ def self.load_file(path)
         #   The optional Certificate Authority (CA) certificate to attach to the
         #   new certificate.
         #
+        # @param [Boolean] ca
+        #   Indicates whether to add the basicConstraints extension.
+        #
+        # @param [Array[String], nil] subject_alt_names
```

Comment:
The syntax should be `Array<String>` (for consistency with the other documentation).